### ISSUE TITLE
Can't install compatible extensions

### ISSUE THREAD

**Issue by polterbyte** [ThemeFuse/Unyson#1691]
Installed Unison 2.5.6 from Wordpress (to help me customize the Journal theme), but if I try to install the compatible extensions, all I get are these messages:

Install Compatible Extensions
Downloading the "Sidebars" extension...

Failed to access Github repository "ThemeFuse/Unyson-Sidebars-Extension" releases.

Downloading the "Portfolio" extension...

Failed to access Github repository "ThemeFuse/Unyson-Portfolio-Extension" releases.

Downloading the "Shortcodes" extension...

Failed to access Github repository "ThemeFuse/Unyson-Shortcodes-Extension" releases.

Downloading the "Backup & Demo Content" extension...

Failed to access Github repository "ThemeFuse/Unyson-Backups-Extension" releases.

Downloading the "Forms" extension...

Failed to access Github repository "ThemeFuse/Unyson-Forms-Extension" releases.

Downloading the "Mailer" extension...

Failed to access Github repository "ThemeFuse/Unyson-Mailer-Extension" releases.

Downloading the "Media" extension...

Failed to access Github repository "ThemeFuse/Unyson-Empty-Extension" releases.

Downloading the "Analytics" extension...

Failed to access Github repository "ThemeFuse/Unyson-Analytics-Extension" releases.

Downloading the "Social" extension...

Failed to access Github repository "ThemeFuse/Unyson-Social-Extension" releases.

Downloading the "Breadcrumbs" extension...

Failed to access Github repository "ThemeFuse/Unyson-Breadcrumbs-Extension" releases.

Failed to access Github repository "ThemeFuse/Unyson-Sidebars-Extension" releases.
Failed to access Github repository "ThemeFuse/Unyson-Portfolio-Extension" releases.
Failed to access Github repository "ThemeFuse/Unyson-Shortcodes-Extension" releases.
Failed to access Github repository "ThemeFuse/Unyson-Backups-Extension" releases.
Failed to access Github repository "ThemeFuse/Unyson-Forms-Extension" releases.
Failed to access Github repository "ThemeFuse/Unyson-Mailer-Extension" releases.
Failed to access Github repository "ThemeFuse/Unyson-Empty-Extension" releases.
Failed to access Github repository "ThemeFuse/Unyson-Analytics-Extension" releases.
Failed to access Github repository "ThemeFuse/Unyson-Social-Extension" releases.
Failed to access Github repository "ThemeFuse/Unyson-Breadcrumbs-Extension" releases.

My site is parked on FeeHostia, on Wordpress 4.5.3.

Thanks!

Ruy

**Comment by moldcraft** [comment]
Hello, please add this code in `wp-content/plugins/unsyon.php`

```php
add_action('admin_notices', function(){
    fw_print(wp_remote_get('http://github-api-cache.unyson.io/'));
});
```

refresh dashboard and give us the screenshot

**Comment by polterbyte** [comment]
OK, did that, but all I get is an error where the Unison dashboard used to be:

<img width="1082" alt="captura de tela 2016-06-29 as 10 38 06" src="https://cloud.githubusercontent.com/assets/20191664/16454117/be13f8fc-3de5-11e6-9e08-83e5e1340ad2.png">

**Comment by moldcraft** [comment]
@polterbyte then try this code

```php
add_action('admin_notices', '_action_test');
function _action_test(){
    fw_print(wp_remote_get('http://github-api-cache.unyson.io/'));
}
```

**Comment by polterbyte** [comment]
OK! Now it seems like we're getting somewhere. Here's the result:

<img width="1264" alt="captura de tela 2016-06-29 as 11 57 12" src="https://cloud.githubusercontent.com/assets/20191664/16457116/b8d6fd02-3df0-11e6-87e6-d07858ccc395.png">

**Comment by moldcraft** [comment]
Your hosting is blocking requests to `http://github-api-cache.unyson.io/`. Please contact your hosting support

**Comment by polterbyte** [comment]
OK! Will do.

Once the blocking is lifted, it should work, right?

Thanks for your support!

**Comment by moldcraft** [comment]
Yes, it should work.

**Comment by polterbyte** [comment]
It didn't work, though...

This is what I get when I try to install the "Sliders" extension

<img width="886" alt="captura de tela 2016-06-29 as 12 21 08" src="https://cloud.githubusercontent.com/assets/20191664/16458072/33189672-3df4-11e6-82db-415b4cb70e09.png">

And this is what I have as a result of those lines of code you instructed me to add:

<img width="1043" alt="captura de tela 2016-06-29 as 12 21 38" src="https://cloud.githubusercontent.com/assets/20191664/16458091/48030478-3df4-11e6-909f-0871f2d6d024.png">

**Comment by moldcraft** [comment]
Test please

```php
add_action('admin_notices', '_action_test');
function _action_test(){
    fw_print(wp_remote_get('https://github.com/ThemeFuse/Unyson-Empty-Extension/archive/v1.0.0.zip'));
}
```

**Comment by polterbyte** [comment]
It worked! It turned out there were several addresses that should be unblocked and after I (painstakingly) did if for all of them, the extensions were downloaded and installed correctly. Thank you very much for your help!

**Comment by catalinafive** [comment]
Hi,
I have the same issue. I use Digital Ocean and they said they don't have that url blocked.
I add the code:
add_action('admin_notices', function(){
			fw_print(wp_remote_get('http://github-api-cache.unyson.io/'));
		});

And this is the result:

array
(
    [headers] => Requests_Utility_CaseInsensitiveDictionary
    (
        [*:data] => array
        (
            [date] => 'Tue, 02 Feb 2021 21:47:35 GMT'
            [server] => 'Apache/2.4.7 (Ubuntu)'
            [x-powered-by] => 'PHP/5.5.9-1ubuntu4.25'
            [cache-control] => 'no-cache'
            [vary] => 'Accept-Encoding'
            [content-encoding] => 'gzip'
            [content-length] => '220'
            [content-type] => 'text/html; charset=UTF-8'
        )
    )
    [body] => '<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
</head>
<body>
<h1 style="text-align: center;">Github API Cache</h1>
</body>
</html>'
    [response] => array
    (
        [code] => 200
        [message] => 'OK'
    )
    [cookies] => array()
    [filename] => null
    [http_response] => WP_HTTP_Requests_Response
    (
        [*:response] => Requests_Response
        (
            [body] => '<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
</head>
<body>
<h1 style="text-align: center;">Github API Cache</h1>
</body>
</html>'
            [raw] => 'HTTP/1.1 200 OK
Date: Tue, 02 Feb 2021 21:47:35 GMT
Server: Apache/2.4.7 (Ubuntu)
X-Powered-By: PHP/5.5.9-1ubuntu4.25
Cache-Control: no-cache
Vary: Accept-Encoding
Content-Encoding: gzip
Content-Length: 220
Connection: close
Content-Type: text/html; charset=UTF-8

<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
</head>
<body>
<h1 style="text-align: center;">Github API Cache</h1>
</body>
</html>'
            [headers] => Requests_Response_Headers
            (
                [*:data] => array
                (
                    [date] => array
                    (
                        [0] => 'Tue, 02 Feb 2021 21:47:35 GMT'
                    )
                    [server] => array
                    (
                        [0] => 'Apache/2.4.7 (Ubuntu)'
                    )
                    [x-powered-by] => array
                    (
                        [0] => 'PHP/5.5.9-1ubuntu4.25'
                    )
                    [cache-control] => array
                    (
                        [0] => 'no-cache'
                    )
                    [vary] => array
                    (
                        [0] => 'Accept-Encoding'
                    )
                    [content-encoding] => array
                    (
                        [0] => 'gzip'
                    )
                    [content-length] => array
                    (
                        [0] => '220'
                    )
                    [content-type] => array
                    (
                        [0] => 'text/html; charset=UTF-8'
                    )
                )
            )
            [status_code] => 200
            [protocol_version] => 1.1
            [success] => true
            [redirects] => 1
            [url] => 'https://github-api-cache.unyson.io/'
            [history] => array
            (
                [0] => Requests_Response
                (
                    [body] => '<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>301 Moved Permanently</title>
</head><body>
<h1>Moved Permanently</h1>
<p>The document has moved <a href="https://github-api-cache.unyson.io/">here</a>.</p>
<hr>
<address>Apache/2.4.7 (Ubuntu) Server at github-api-cache.unyson.io Port 80</address>
</body></html>
'
                    [raw] => 'HTTP/1.1 301 Moved Permanently
Date: Tue, 02 Feb 2021 21:47:35 GMT
Server: Apache/2.4.7 (Ubuntu)
Location: https://github-api-cache.unyson.io/
Content-Length: 334
Connection: close
Content-Type: text/html; charset=iso-8859-1

<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>301 Moved Permanently</title>
</head><body>
<h1>Moved Permanently</h1>
<p>The document has moved <a href="https://github-api-cache.unyson.io/">here</a>.</p>
<hr>
<address>Apache/2.4.7 (Ubuntu) Server at github-api-cache.unyson.io Port 80</address>
</body></html>
'
                    [headers] => Requests_Response_Headers
                    (
                        [*:data] => array
                        (
                            [date] => array
                            (
                                [0] => 'Tue, 02 Feb 2021 21:47:35 GMT'
                            )
                            [server] => array
                            (
                                [0] => 'Apache/2.4.7 (Ubuntu)'
                            )
                            [location] => array
                            (
                                [0] => 'https://github-api-cache.unyson.io/'
                            )
                            [content-length] => array
                            (
                                [0] => '334'
                            )
                            [content-type] => array
                            (
                                [0] => 'text/html; charset=iso-8859-1'
                            )
                        )
                    )
                    [status_code] => 301
                    [protocol_version] => 1.1
                    [success] => false
                    [redirects] => 0
                    [url] => 'http://github-api-cache.unyson.io/'
                    [history] => array()
                    [cookies] => Requests_Cookie_Jar
                    (
                        [*:cookies] => array()
                    )
                )
            )
            [cookies] => Requests_Cookie_Jar(...)
        )
        [*:filename] => null
        [data] => null
        [headers] => null
        [status] => null
    )
)

What should I do? Thanks
